tests/pom.xml

```diff
@@ -228,6 +244,55 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
               <autoCreateCustomNetworks>true</autoCreateCustomNetworks>
               <startParallel>false</startParallel>
               <images>
+                <!-- ##### Mongo DB instance for the device registry ##### -->
+                <image>
+                  <name>${docker.image.org-name}/hono-mongodb-test</name>
+                  <alias>mongo</alias>
+                  <build>
+                    <skip>${hono.mongodb.disabled}</skip>
+                    <imagePullPolicy>IfNotPresent</imagePullPolicy>
+                    <from>${hono.mongodb.image.name}</from>
+                    <assembly>
+                      <mode>dir</mode>
+                      <basedir>/</basedir>
+                      <inline>
+                        <id>config</id>
+                          <fileSet>
+                            <directory>${project.build.directory}/resources/mongodb/init_db</directory>
+                            <outputDirectory>docker-entrypoint-initdb.d/</outputDirectory>
+                            <includes>
+                              <include>*</include>
+                            </includes>
+                          </fileSet>
+                      </inline>
+                    </assembly>
+                  </build>
+                  <run>
+                    <skip>${hono.mongodb.disabled}</skip>
+                    <ports>
+                      <port>+mongodb.ip:mongodb.port:${hono.mongodb.port}</port>
+                    </ports>
+                    <portPropertyFile>${project.build.directory}/docker/mongodb.port.properties</portPropertyFile>
+                    <env>
+                      <MONGO_INITDB_DATABASE>${hono.mongodb.database.name}</MONGO_INITDB_DATABASE>
+                    </env>
+                    <network>
+                      <mode>custom</mode>
+                      <name>hono</name>
+                      <alias>${hono.mongodb.host}</alias>
+                    </network>
+                    <memorySwap>314572800</memorySwap>
+                    <memory>314572800</memory>
+                    <log>
+                      <prefix>MONGODB</prefix>
+                      <color>green</color>
```

Comment:
please use `${log.color.hono-device-registry}` instead

Reply:
I have updated it.